lib/Service/Gateway/Signal/Gateway.php

```diff
@@ -62,21 +62,41 @@ public function __construct(IClientService $clientService,
 	 */
 	public function send(IUser $user, string $identifier, string $message) {
 		$client = $this->clientService->newClient();
-		$response = $client->post(
-			$this->config->getUrl(),
-			[
-				'body' => [
-					'to' => $identifier,
-					'message' => $message,
-				],
-			]
-		);
-		$body = $response->getBody();
-		$json = json_decode($body, true);
+		// determine type of gateway
+		$response = $client->get($this->config->getUrl() . '/v1/about');
+		if ($response->getStatusCode() === 200) {
+			// New style gateway https://gitlab.com/morph027/signal-cli-dbus-rest-api
```

Comment:
Is this comment necessary?

Reply:
What is a "necessary" comment?

Reply:
Ok, this show be further discussed here: #481